src/gsi/read_iasi.f90

```diff
@@ -827,7 +827,11 @@ subroutine read_iasi(mype,val_iasi,ithin,isfcalc,rmesh,jsatid,gstime,&
 !          Put satinfo defined channel temperatures into data array
            do l=1,satinfo_nchan
               i = bufr_index(l)
-              data_all(l+nreal,itx) = temperature(i)   ! brightness temerature
+              if(i /= 0)then
+                 data_all(l+nreal,itx) = temperature(i)   ! brightness temerature
+              else
```

Comment:
Should we put a warning printout here to indicate i=0  and tbmin is used?

Reply:
I think I convinced myself that the data would not be used if i=0.  So I don't think any print out is necessary.

Reply:
Yes.  In the qc routines if the data is equal to tbmin, the obs errors are set so that it will not be used.